BeVolt/BSP/Simulator/DataGeneration/WDTimer.py

```diff
@@ -18,16 +21,16 @@ def Check_State():
             fcntl.flock(csvfile.fileno(), fcntl.LOCK_EX)  
             csvreader = csv.reader(csvfile)
             currentState = next(csvreader)
-            if ((currentState[0] == 'aaaa') and (duration > 0)):
+            if ((currentState[0] == reset) and (duration > 0)):
                 duration = 15 #reset countdown
                 csvfile.seek(0)
                 csvwriter = csv.writer(csvfile)
```

Comment:
csvfile is opened with `'r+'`, which will fail if the file is not there. Probably should change this to `'w+'`. Placing the comment here since Github won't let me comment on the relevant line. Look at line 20.